Source/textdat.cpp

```diff
@@ -735,6 +735,74 @@ const TextDataStruct alltext[] = {
+
+	    true, TSFX_SMITH16 },
+	{ "I'm in luck! A caravan has stopped just outside of the village and is taking supplies to the lands of the East. Certain items will bring a special price... if you can get them for me. |",
+
+	    true, TSFX_SMITH27 },
+	{ "We can both turn a nice profit if you can deliver the right goods to me. What do you say? |",
+
+	    true, TSFX_SMITH28 },
+	{ You're just in time. The caravan is leaving tonight! Here's your cut, friend. |",
+
+	    true, TSFX_SMITH29 },
+	{ "I believe that I may have found a way to greatly improve some weapons. If you can bring me what I need, I'd be willing to try out my idea and, if it works, it's yours! |",
+
+	    true, TSFX_SMITH30 },
+	{ "I still think I can improve the right weapon! |",
+	    true, TSFX_SMITH31 },
+	{ "Let me just do this... and this... and this. Ah, it works! Take this back and give it a try. |",
+
+	    true, TSFX_SMITH32 },
+	{ "I'm working on a method for strengthening armor. Unfortunately, I don't have what I need just now. If you could possibly bring me what I require, the first one is yours! |",
+
+	    true, TSFX_SMITH33 },
+	{ "I still think I can strengthen the right armor. You know what you need to bring me. |",
+	    true, TSFX_SMITH34 },
+	{ "Good find! See if this works any better for you. |",
+	    true, TSFX_SMITH35 },
+	{ "If you come across any enchanted equipment, I could try to learn how it was crafted and then use those methods for my own creations. I should probably start with something easy. |",
+
+	    true, TSFX_SMITH36 },
+	{ "I still think that I can strip the knowledge of enchantments... if you bring me a certain item. |",
+
+	    true, TSFX_SMITH37 },
+	{ "Thank you, oh great and mighty customer... err... uhh... champion! This will take some study... Done! |",
+
+	    true, TSFX_SMITH38 },
+	{ "Keep your eyes open for a figurine made of iron. I have uncovered some old records concerning a specific type of figurine and the secrets of metalcrafting that it holds. |",
+
+	    true, TSFX_SMITH39 },
+	{ "I know that the right figurine could be quite powerful. If you see what I'm looking for, get it and bring it to me straight away! |",
+
+	    true, TSFX_SMITH40 },
+	{ "The secrets of this metal are fantastic! Oh, how I could make the right item gleam with power. |",
+
+	    true, TSFX_SMITH41 },
+	{ "If you deliver to me the right item so I can combine it with this metal, I could craft something worthy of the warriors of Heaven! |",
+
+	    true, TSFX_SMITH42 },
+	{ "Your efforts are not in vain. It seems that all is as I had hoped. I trust you will find this useful in your battles below? |",
+
+	    true, TSFX_SMITH43 },
 */
 };
 

```

Comment:
```suggestion
	{ "I've seen no such things in my shop, but I will keep a close watch for them. Perhaps if they come this way, they won't take kindly to the fires of my forge. |",
	    true, TSFX_SMITH5 },
	{ "I admit that I too have seen these strange lights, but I have not felt pulled towards them. I'll be sure to steer clear of them and will tell everyone that visits here to do likewise. |",
	    true, TSFX_SMITH6 },
	{ "A bold tale indeed! My limited time beneath the cathedral leaves me poorly equipped to offer you any help with this. But, as always, you'll find Cain well versed in legends and folklore. |",
	    true, TSFX_SMITH8 },
	{ "The blade Azurewrath... It's legend! It was cast by the angelic weaponsmith Cinadide and tempered within the fires of judgment! Whoever wields this weapon will find the legions of Hell at his feet! If you found this blade, I would begin to truly believe that you could end the nightmare that has befallen our town. |",
	    true, TSFX_SMITH9 },
	{ "When I found Wirt, he was very near a room that sounds like the vile pit you've described. The stench of death was heavy in the air and, consumed as I was with getting the lad to safety, I did not go further than I had to. If he claims to know something of that place, I would not discount his word easily. |",
	    true, TSFX_SMITH11 },
	{ "Farnham speaks of a place that exists, at least in legend. Warriors would go to a place at the edge of Hell to gird themselves for battle against the armies of darkness. If the stories are true, untold treasures could lie upon this island of the sunless sea. |",
	    true, TSFX_SMITH15 },
	{ "Ah, you speak of an ancient and evil weapon. Tread lightly in this area, for the legends of Shadowfang are as black as a moonless winter night. Crafted within the Hellforge, Shadowfang can rend the very soul from whoever it strikes. I do not envy you if is in your mind to defeat the one who wields it. May light protect you, brave hero. |",
	    true, TSFX_SMITH16 },
	{ "I'm in luck! A caravan has stopped just outside of the village and is taking supplies to the lands of the East. Certain items will bring a special price... if you can get them for me. |",
	    true, TSFX_SMITH27 },
	{ "We can both turn a nice profit if you can deliver the right goods to me. What do you say? |",
	    true, TSFX_SMITH28 },
	{ You're just in time. The caravan is leaving tonight! Here's your cut, friend. |",
	    true, TSFX_SMITH29 },
	{ "I believe that I may have found a way to greatly improve some weapons. If you can bring me what I need, I'd be willing to try out my idea and, if it works, it's yours! |",
	    true, TSFX_SMITH30 },
	{ "I still think I can improve the right weapon! |",
	    true, TSFX_SMITH31 },
	{ "Let me just do this... and this... and this. Ah, it works! Take this back and give it a try. |",
	    true, TSFX_SMITH32 },
	{ "I'm working on a method for strengthening armor. Unfortunately, I don't have what I need just now. If you could possibly bring me what I require, the first one is yours! |",
	    true, TSFX_SMITH33 },
	{ "I still think I can strengthen the right armor. You know what you need to bring me. |",
	    true, TSFX_SMITH34 },
	{ "Good find! See if this works any better for you. |",
	    true, TSFX_SMITH35 },
	{ "If you come across any enchanted equipment, I could try to learn how it was crafted and then use those methods for my own creations. I should probably start with something easy. |",
	    true, TSFX_SMITH36 },
	{ "I still think that I can strip the knowledge of enchantments... if you bring me a certain item. |",
	    true, TSFX_SMITH37 },
	{ "Thank you, oh great and mighty customer... err... uhh... champion! This will take some study... Done! |",
	    true, TSFX_SMITH38 },
	{ "Keep your eyes open for a figurine made of iron. I have uncovered some old records concerning a specific type of figurine and the secrets of metalcrafting that it holds. |",
	    true, TSFX_SMITH39 },
	{ "I know that the right figurine could be quite powerful. If you see what I'm looking for, get it and bring it to me straight away! |",
	    true, TSFX_SMITH40 },
	{ "The secrets of this metal are fantastic! Oh, how I could make the right item gleam with power. |",
	    true, TSFX_SMITH41 },
	{ "If you deliver to me the right item so I can combine it with this metal, I could craft something worthy of the warriors of Heaven! |",
	    true, TSFX_SMITH42 },
	{ "Your efforts are not in vain. It seems that all is as I had hoped. I trust you will find this useful in your battles below? |",
	    true, TSFX_SMITH43 },
*/
};

```